account_financial_report_qweb/static/src/css/report.css

```diff
@@ -96,11 +97,12 @@
 }
 
 .o_account_financial_reports_page {
-    background-color: @odoo-view-background-color;
+    background-color: white;
```

Comment:
Isn't a variable for enterprise background? I still prefer to keep "transparent" background

Reply:
this is broken anyway, is not LESS ;)

Reply:
we should move the whole file to LESS and then play w/ the variables

Reply:
My question was:

> Isn't a variable for enterprise background?

and my purpose is to let it transparent, but in both versions.